eng/.docsettings.yml

```diff
@@ -70,6 +70,7 @@ known_presence_issues:
   - ['sdk/keyvault','#5499']
   - ['sdk/eventhub','#5499']
   - ['sdk/attestation/Microsoft.Azure.Attestation','#5499']
+  - ['sdk/storage/Azure.Storage.Blobs.AspNetCore.DataProtection','#9960']
```

Comment:
I think most of these bugs reference the `azure-sdk-tools` repo so it would be good to make this something more like `azure-sdk-for-net #9960`.

Reply:
No, they are all in azure-sdk-for-net

Reply:
Sorry, I had it [backwards](https://github.com/Azure/azure-sdk-for-net/blob/e97701b729ebf54c4095c1af57c887695b32a0fc/eng/.docsettings.yml#L133).  You're correct.